examples/powertools-examples-cloudformation/infra/cdk/src/main/java/com/myorg/PowertoolsExamplesCloudformationCdkApp.java

```diff
@@ -0,0 +1,16 @@
+package com.myorg;
```

Comment:
I reckon we should tie this CDK project (and any future ones) into the build, but not the main build job. We want to know if these break, but we'd like them to run in // to the rest of the build and I think with a degree of logical separation.